### DISCUSSION TITLE
Improving the Read aloud feature

### DISCUSSION THREAD

**Discussion by Sagar0-0** [kiwix/kiwix-android#3750] [Ideas]
We can improve the read-aloud feature UI and functionality. In terms of UI, we can give a cleaner pause and stop buttons UI and in terms of functionality, I believe if we can give the option to start reading from wherever the user wants, then it will be very efficient and time saving. If it is possible for us to fetch the user selection in a zim file, then maybe we can make it possible

**Comment by kelson42** [comment]
@Sagar0-0 I would really recommend here to make a proposal if you have a clear idea how to improve it
